resources/_gen/assets/js/js/search.js_b4bb3e7b21b43e920b7c367ff787d383.content

```diff
@@ -27,7 +27,7 @@ limitations under the License.
                     }
 
                     var query = $(this).val();
-                    var searchPage = "search/?q=" + query;
+                    var searchPage = "//localhost:1313/search/?q=" + query;
```

Comment:
Should this `localhost` be here?

Reply:
Thanks for flagging Greg, I hadn't spotted this. That is strange, I didn't change the config. Everything under /resources is generated by the `hugo` command. so I'm not what action I did that caused this to update. I'm not too worried currently as search is disabled and I wasn't planning on enabling it before launch.

Anyway I've reverted the changes to this file and the site still builds as expected. Preview at:

https://5c75a630f330ec00083972e2--elated-haibt-1b47ff.netlify.com/

Logs viewable at:

https://app.netlify.com/sites/elated-haibt-1b47ff/deploys/

Thanks for the review!